app/genGraphData.ts

```diff
@@ -79,7 +79,14 @@ export const makeNodesAndEdges = (data: NormalizedData): [cytoscape.ElementDefin
             shape: "roundrectangle",
             "text-valign": "top",
         })),
-        edgeStyle("all", css({ opacity: 0.3 })),
+        edgeStyle("all", css({ 
+            opacity: 0.3,
+            width: function(e) {
+                if (e.data("main") === true) return "6px";

```

Comment:
This is awesome, @jollypolly123 !

One thing to note: As part of the 'normalization' process to the data before it's written to `data.json`, I process all the project connections so that they have a `weight` (if one was not specified, then a default value is used based on whether `main` was specified or not). 

Here's the code that does that: https://github.com/mitmedialab/PRG-Group-Map/blob/main/builder/index.ts#L133

Were you finding that this code is necessary? If not, I'd try getting rid of it and just assuming that the `weight` value is present.

But if that causes issues, then changes should be made on the backend to ensure this isn't necessary.

That way we can avoid all this data cleaning/checking on the front end side.

Reply:
Gotcha! Just moved it to the backend. I changed the scale of the weight to be in px so we don't have to do any processing.